src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 # Required for building the dashboard grpc protos at build time
 set(CEPH_GRPC_VIRTUALENV ${CEPH_BUILD_VIRTUALENV}/ceph-grpc-virtualenv)
 set(CEPH_NVME_GATEWAY_PROTO_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../src/nvmeof/gateway/control/proto)
+set(GRPC_TOOLS_DIR ${CEPH_GRPC_VIRTUALENV}/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/grpc_tools)
+
+message(STATUS "install grpc-tools into ${GRPC_TOOLS_DIR}")
```

Comment:
The truth is that we don't even need to handle this (CC: @pecastro). The installation of `grpc*` stuff should be done via `BuildRequires`/Debian-equivalent. And this tool is invoked through `python3 -m grpcio_tools.protoc`, so same thing.